### PR TITLE
include colors with alpha values fixes #20

### LINKED ISSUE #20: Grayscale fill() or background() with alpha value crashes sketch
#### Nature of issue?

- [x] Found a bug
- [ ] Existing feature enhancement
- [ ] New feature request

#### Details about the bug: 
Originally from processing/p5.js-web-editor#590

- Web browser and version: 64.0.3282.186 (Official Build) (64-bit) (cohort: Stable)
- Operating System: Windows 10 1709
- Steps to reproduce this:

```javascript
function setup() {
  createCanvas(400, 400);
}

function draw() {
  background(0, 10);
}
```

```javascript
function setup() {
  createCanvas(400, 400);
}

function draw() {
  background(0);
  fill(255, 255)
  ellipse(mouseX, mouseY, 10, 10)
}
```
After a few frames, the sketch stops working (it freezes) and I get `Script error. (: line 0)` in the console. This only happens in Editor mode, in Fullscreen mode, it works fine. This even happens when the alpha value is 255 or 0.

The following error appears in the JS console:
```
Uncaught TypeError: Cannot read property 'color' of undefined
    at BackgroundEntity.populate (https://cdn.rawgit.com/MathuraMG/p5-accessibility/298ccdf6/dist/p5-accessibility.js:1059:68)
    at new BackgroundEntity (https://cdn.rawgit.com/MathuraMG/p5-accessibility/298ccdf6/dist/p5-accessibility.js:1062:8)
    at gridInterceptor.populateObject (https://cdn.rawgit.com/MathuraMG/p5-accessibility/298ccdf6/dist/p5-accessibility.js:1524:5)
    at e.p5.(anonymous function) [as background] (https://cdn.rawgit.com/MathuraMG/p5-accessibility/298ccdf6/dist/p5-accessibility.js:1658:25)
    at draw (about:srcdoc:55:3)
    at e.redraw (https://cdnjs.cloudflare.com/ajax/libs/p5.js/0.6.0/p5.min.js:8:27245)
    at e.p5.(anonymous function) [as redraw] (https://cdn.rawgit.com/MathuraMG/p5-accessibility/298ccdf6/dist/p5-accessibility.js:1673:27)
    at e.<anonymous> (https://cdnjs.cloudflare.com/ajax/libs/p5.js/0.6.0/p5.min.js:7:17856)
```

### PR DESCRIPTION
**Pull request by lm-n**
This PR fixes #20 by including colors with alpha values. It includes:
- grayscale + alpha
- RGBA
There's other issues to fix with regards to colors but I think it might be a good idea to go ahead with this PR and then look into the other issues. I can do that tomorrow. I'll make a new issue for those other color problems #36.

### CODE REVIEW

**Comment by MathuraMG** [comment]
Looks good @lm-n !
Just a quick language thing - if it's 0% transparent, should we say "solid" (or not say it at all) ? So that the sentence doesn't become unnecessarily longer.

**Comment by lm-n** [comment]
I think not say it at all is good! Just made the change.

**Comment by lm-n** [comment]
Just solved the other color issues from #36 only one missing is when using colorMode(). @MathuraMG could you take a look? and let me know if we should merge?

**Comment by MathuraMG** [comment]
fill('rgba(100%,30%,2%)'); is valid, but doesn't work with the library

**Comment by lm-n** [comment]
Yes I noticed that too. There are certain values like "fill('rgb(100%,0%,10%)')" are not supported by the library

**Comment by MathuraMG** [comment]
Wohoo! This doesn't say 0% transparency anymore -> 'rgba(100%,30%,2%,1)'

**Comment by lm-n** [comment]
@MathuraMG fixed the other error

File: src/baseInterceptor.js [hunk 1]
```diff
@@ -182,7 +182,7 @@ function RGBAString(arguments){
         }
       }
       return(getRGBAname(values));
-    }else{
+    }else if (((arguments[0].match(/%/g)).length)==3){
       //when arguments[0] is 'rgba(10%,100%,30%,0.5)'
```
Comment (MathuraMG) [inline src/baseInterceptor.js]: Can we remove the last argument?  //when arguments[0] is 'rgba(10%,100%,30%,0.5)'

**Pull request merged**